commands/databasescommands/backupcommands/create.go

```diff
@@ -0,0 +1,114 @@
+package backupcommands
+
+import (
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/db/v1/backups"
+	"github.com/rackspace/rack/util"
+)
+
+var create = cli.Command{
+	Name:        "create",
+	Usage:       util.Usage(commandPrefix, "create", "--instance <instanceId> --name <name> --description <description>"),
```

Comment:
The reason for the `--instance-id` change would be if we'd potentially allow a `--instance-name` parameter instead, where we'd lookup the ID by name. Is such a thing possible in the Databases service?